integration-tests/container/so_reuseport/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:jammy
+
+COPY so_reuseport.py so_reuseport.py
+
+RUN apt update -y && apt install python3 -y
+
+ENTRYPOINT python3 so_reuseport.py & python3 so_reuseport.py meaningless_argument
```

Comment:
Note that without `meaningless_argument` only one listening endpoint is reported. Perhaps there should also be a test for this case.